### PR TITLE
Update recommended Xcode settings

### PR DESCRIPTION
**Pull request by ibrahimkteish**
Silent 5 warnings : "Result of call to 'map' is unused"

### CODE REVIEW

File: CryptoSwift/SHA2.swift [hunk 1]
```diff
@@ -124,7 +124,7 @@ final class SHA2 : HashProtocol {
         
         // hash values
         var hh = [UInt32]()
-        variant.h.map({(h) -> () in
+        _ = variant.h.map({(h) -> () in
```
Comment (krzyzanowskim) [inline CryptoSwift/SHA2.swift]: forEach() is better match here

Comment (ibrahimkteish) [inline CryptoSwift/SHA2.swift]: Done :)

File: CryptoSwift/SHA2.swift [hunk 2]
```diff
@@ -124,10 +124,10 @@ final class SHA2 : HashProtocol {
         
         // hash values
         var hh = [UInt32]()
-        variant.h.map({(h) -> () in
-            hh.append(UInt32(h))
-        })
-        
+		variant.h.forEach {(h) -> () in
```
Comment (krzyzanowskim) [inline CryptoSwift/SHA2.swift]: awesome, what happened to the indentation here ? :)

Comment (ibrahimkteish) [inline CryptoSwift/SHA2.swift]: it doesn't appear like this in Xcode

Comment (krzyzanowskim) [inline CryptoSwift/SHA2.swift]: looks you have tabs in place of spaces.

Comment (ibrahimkteish) [inline CryptoSwift/SHA2.swift]: what should I do ?

Comment (krzyzanowskim) [inline CryptoSwift/SHA2.swift]: screen shot 2015-08-31 at 12 51 27

**Comment by krzyzanowskim** [comment]
@ibrahimkteish thanks for the PR. I want to merge it, just fix the indentation everywhere: https://github.com/krzyzanowskim/CryptoSwift/pull/119/files

**Comment by ibrahimkteish** [comment]
Done :)

File: CryptoSwift.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -664,9 +666,11 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LLVM_LTO = YES;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.hakore.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_OPTIMIZATION_LEVEL = "-Ounchecked";
+				SWIFT_DISABLE_SAFETY_CHECKS = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
```
Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: I missed this change... I don't want it -O, it suppose to be -Ounchecked due to performance reasons ;_

Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: Now I see how thing changed in Xcode7

Comment (ibrahimkteish) [inline CryptoSwift.xcodeproj/project.pbxproj]: Do you think
SWIFT_DISABLE_SAFETY_CHECKS = YES;			SWIFT_OPTIMIZATION_LEVEL = "-O";
Replace

SWIFT_OPTIMIZATION_LEVEL = "-Ounchecked";

**Pull request merged**